test/commands/credentials.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+/* global describe it beforeEach afterEach */
+
+const cli = require('heroku-cli-util')
+const expect = require('unexpected')
+const nock = require('nock')
+const proxyquire = require('proxyquire')
+
+const db = {
+  database: 'mydb',
+  host: 'foo.com',
+  user: 'jeff',
+  password: 'pass',
+  url: {href: 'postgres://foo'}
```

Comment:
Not a big deal, but I think internal consistency in the test data would be nice, so `{href: 'postgres://jeff:pass@foo.com/mydb'}`...